### PR TITLE
Fix web window manager

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

File: core/base/src/TQObject.cxx [hunk 1]
```diff
@@ -1011,7 +1011,7 @@ Bool_t TQObject::Disconnect(const char *class_name,
    TClass *sender = TClass::GetClass(class_name);
 
    // sender should be TQClass (which derives from TQObject)
-   if (!sender->IsA()->InheritsFrom(TQObject::Class()))
+   if (!sender || !sender->IsA()->InheritsFrom(TQObject::Class()))
```
Comment (linev) [inline core/base/src/TQObject.cxx]: Why sender->IsA() used here?
sender is already pointer on the class, one should just check sender->InheritsFrom(TQObject::Class())

Comment (bellenot) [reply core/base/src/TQObject.cxx]: Well, it's your code...

File: gui/webdisplay/src/RWebWindowsManager.cxx [hunk 2]
```diff
@@ -472,7 +474,6 @@ void ROOT::Experimental::RWebWindowsManager::Terminate()
    if (fServer)
       fServer->SetTerminate();
 
-   // use timer to avoid situation when calling object is deleted by terminate
```
Comment (bellenot) [inline gui/webdisplay/src/RWebWindowsManager.cxx]: @linev I don't understand why you are using a timer here. Can you elaborate?

Comment (linev) [reply gui/webdisplay/src/RWebWindowsManager.cxx]: I have doubt, that gApplication->Terminate() may call destructor of any object - including RWebWindowManager itself. But if direct call works better - just use it

**Pull request merged**